### ISSUE TITLE
[Feature request] Add XMind to Flathub.org

### ISSUE THREAD

**Issue by dsboger** [xmindltd/xmind#312]
It would be great if you could add XMind to flathub. It would mean writing and maintaining a flatkpak manifest, but then XMind would be universally available on every Linux distro.

Some documentation that might be of help:

https://github.com/flathub/flathub/wiki/App-Submission
https://github.com/flathub/flathub/wiki/App-Requirements

Thanks!

**Comment by muelli** [comment]
yeah, I have a *very* dirty flatpak-builder manifest that basically unzips the released zip file into the home directory, because somehow I am unable to make it work from `/app/`. Supposedly because the user cannot write there.

Notes to my future self: `XMind_amd64/XMind` wants to be called with the cwd set to `XMind_amd64`. Calling it from a different working directory did not work for me.

Also JRE9 did not work for me. Unfortunately, jdk8 is not readily available as a flatpak extension, it seems.

I have yet to understand `xmind.ini`, `configuration.ini`, and parameters given to the XMind executable.

It's a very weird package, that released zip file.

**Comment by jacknlliu** [comment]
+1

**Comment by chrisweeksnz** [comment]
It would be wonderful to see a flatpak of XMind for distributions that have strict app acceptance rules (Solus Linux) and containerised systems (Fedora Silverblue)!

**Comment by muelli** [comment]
cf. https://github.com/flathub/flathub/pull/889 and https://github.com/flathub/flathub/pull/892

**Comment by nedrichards** [comment]
We've added those apps to flathub. If you'd like to take over the repo to maintain the version there then let us know and we'll be happy to help.
